azure_pipelines/spm-framework.yml

```diff
@@ -11,6 +11,8 @@ variables:
     value: 'master'
```

Comment:
This pull request does not update CHANGELOG.md.

Please consider if this change would be noticeable to a partner or user and either update CHANGELOG.md or resolve this conversation.
<!-- changelog-comment -->